### PR TITLE
lazy-loading-in-html-images

### PR DESCRIPTION
**Pull request by Alidhuniya**

### CODE REVIEW

**Comment by atapas** [comment]
@Alidhuniya Which issue is it related to? Please provide the issue link here.

**Comment by Alidhuniya** [comment]
> @Alidhuniya Which issue is it related to? Please provide the issue link here.

sorry, you don't have contribution guide. Here is the issue link. Issue No, (#18)

**Review by atapas — COMMENTED** [review]
@Alidhuniya You got to modify the main index file to link this demo.

**Comment by Alidhuniya** [comment]
> @Alidhuniya You got to modify the main index file to link this demo.

it's linked now

File: lazy-loading/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lazy Loading</title>
+</head>
+<body>
+    <h1>Lazy-Loading-with-html</h1>
+    <h2>Open Network tab and scroll at the same time t see effect</h2>
```
Comment (atapas) [inline lazy-loading/index.html:11]: @Alidhuniya Typo

image

**Review by atapas — CHANGES_REQUESTED** [review]
Please correct the typo.

File: lazy-loading/index.html (hunk 1 above)
Comment (Alidhuniya) [reply lazy-loading/index.html]: > 

fixed..have a look at

**Review by atapas — APPROVED** [review]
@Alidhuniya Thanks for your contribution. Looks good.

**Pull request merged**